src/main/resources/label-definitions.properties

```diff
@@ -1016,7 +1017,9 @@ vessel=upload android
 view-job-filters=ui scm-related maven user emailext
 viewVC=scm-related
 violation-columns=listview-column
-violation-comments-to-stash=maven dotnet scm-related scm report post-build
+violation-comments-to-github=github report post-build analysis devops
+violation-comments-to-gitlab=gitlab report post-build analysis devops
+violation-comments-to-stash=stash bitbucket-server bitbucket report post-build analysis devops
```

Comment:
These will result in a bunch of unsupported labels when displayed in core, but doesn't hurt.